### PR TITLE
Fix usage of scoped delegate in the runTask() API

### PR DESCRIPTION
**Pull request by AndrejMitrovic**

### CODE REVIEW

**Comment by Geod24** [comment]
WIP?

**Comment by AndrejMitrovic** [comment]
Sorry I accidentally added `version (none):` to the other tests. Fixed. 😇

File: source/geod24/LocalRest.d [hunk 1]
```diff
@@ -931,6 +931,45 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
         }
 }
 
+/// test for scoped delegates
+unittest
```
Comment (Geod24) [inline source/geod24/LocalRest.d]: Can you move this towards the end please ? The first example is meant to be / stay a simple usage example. Also I wouldn't make this a documented test

**Comment by AndrejMitrovic** [comment]
Good idea to move it elsewhere. And we should probably move the tests to another module as LocalRest.d is getting quite big.

**Comment by AndrejMitrovic** [comment]
done

**Pull request merged**